tide/wall/TextureNodeYUV.cpp

```diff
@@ -49,11 +49,11 @@ namespace
 
 const char* vertShader =
 R"(
-#version 130
+#version 120
```

Comment:
How are you creating your OpenGL context?
Version 120 corresponds to OpenGL 2. In some cases you can use version 130 GLSL and it may work, but I think it has undefined behaviour if your context is not GL3.